packages/components/src/drop-zone/index.tsx

```diff
@@ -181,20 +109,31 @@ export function DropZoneComponent( {
 			setIsDraggingOverElement( false );
 		},
 	} );
+
+	const shouldActivate =
+		( isDraggingOverDocument || isDraggingOverElement ) &&
+		( ( type === 'file' && onFilesDrop ) ||
+			( type === 'html' && onHTMLDrop ) ||
+			( type === 'default' && onDrop ) );
+
 	const classes = clsx( 'components-drop-zone', className, {
-		'is-active':
-			( isDraggingOverDocument || isDraggingOverElement ) &&
-			( ( type === 'file' && onFilesDrop ) ||
-				( type === 'html' && onHTMLDrop ) ||
-				( type === 'default' && onDrop ) ),
-		'is-dragging-over-document': isDraggingOverDocument,
```

Comment:
I've removed these classes as I couldn't find where they were used. They don't seem to be needed for the component to function. If you know they are used somewhere, let me know, and I'll add them back.

Reply:
Added back those classnames

Reply:
`is-dragging-over-document` could possibly be unused, I didn't spot any usage when I was doing my wpdirectory.net search. Sounds good as a follow-up cleanup too.